shared/notifications/index.js

```diff
@@ -0,0 +1,94 @@
+import { getSwapsFeatureLiveness } from '../../ui/app/ducks/swaps/swaps';
+import { getSwapsEthToken, getIsMainnet } from '../../ui/app/selectors';
```

Comment:
Generally we shouldn't import anything from the `ui` or `app` directories from this `shared` directory. The purpose of `shared` was to have a place for code used in both the UI and the background. Importing UI modules here will include them in the background bundle, even if they aren't called at runtime.

I'd suggest moving the selectors into the UI.

Reply:
> I'd suggest moving the selectors into the UI.

But these selectors are being called in this file. Are you suggesting that the functions that are calling them should also be moved into the UI?

Reply:
I think it might be cool to have shared selectors, when we have typescript and our state trees are strongly typed. It's a little dangerous, though, to have a 'state' parameter in a function that is shared because in background that might be a slightly different shape.

Recommendation: call selectors in ui/ background/ and pass the underlying values to the functions that are shared.

Reply:
Yes, exactly. When I said "selectors" I meant to refer to these functions actually, as generally I think of any function that derives something from selectors as being a selector itself as well.

Reply:
Shared selectors would indeed be useful in many cases, but less so here because these are never used in the background. These functions are called solely in the UI.

Reply:
Oh then they definitely don't need to be in shared/

Reply:
Issue discussed above is resolved in 16ff6e67c